### ISSUE TITLE
Timezone and Startup

### ISSUE THREAD

**Issue by cybercon67** [zombieFox/nightTab#231]
Hi, 
Night Tab is awsome. Have some issues thoug. Im in Thailand, my computer clock is set to Thai time, but the clock in Night Tab shows what seems for me European times. Right now time is 10:43 in Thailand, in Night Tab it is 03:43
Is it some setings Im missing ?

And, ending a session. Closing the browser, opening it again takes me to the default settings, not my modified settings (startpage) tabs work fine

**Comment by zombieFox** [comment]
Hi

Can you provide a few more details please?

- What browser (and the version) are you using?
- What OS (and the version) are you using?
- Do you have any other extensions installed which clears cache or cookies on restart?

Please see these issues regarding the time:
https://github.com/zombieFox/nightTab/issues/105

And data loss when closing the browser:
https://github.com/zombieFox/nightTab/issues/6#issuecomment-453029437

**Comment by Useless-Panda** [comment]
Same Issue my time on windows is 1:48 but time shown on 20:49, i'm using librewolf
can you add option to change time

**Comment by tramabruta** [comment]
I'm also using librewolf which enables RFP I wonder if there is a way to fix this without disabling it...

**Comment by ExpertMen** [comment]
Yep, I am using LibreWolf......huh, how about to "whitelist" nightTab? You can do it by going to Settings > Privacy and Security and there must be a setting for except of clearing cookies of websites.
